.eslintignore

```diff
@@ -1,4 +1,4 @@
 src/sidebar/kinto-http.js
 src/sidebar/quill.js
```

Comment:
Not sure if these lines are needed anymore now that you've deleted from repo and moved into "src/sidebar/vendor/*".